src/jquery.mockjax.js

```diff
@@ -269,6 +269,7 @@
 				url: mockHandler.proxy,
 				type: mockHandler.proxyType,
 				data: mockHandler.data,
+				async: false,
```

Comment:
When it is synchronous and with response time with a proxy the code gets bit messy with callbacks.
Since the code doesnt have a blocking mechanism for synchronous calls, code get even messy.